packages/react/src/components/LocaleModal/LocaleModal.js

```diff
@@ -114,7 +114,7 @@ const LocaleModal = ({ isOpen, setIsOpen, localeData, localeDisplay }) => {
         <ModalHeader
           data-autoid={`${stablePrefix}--locale-modal__region-back`}
           label={[
-            <LinkWithIcon href="#" iconPlacement={'left'}>
+            <LinkWithIcon href="javascript:void(null);" iconPlacement={'left'}>
```

Comment:
Hey @IgnacioBecerra I think React throws console errors for `javascript:void(null)`

> Warning: A future version of React will block javascript: URLs as a security precaution. Use event handlers instead if you can. If you need to generate unsafe HTML try using dangerouslySetInnerHTML instead. React was passed "javascript:;".

You might want to switch to event handler instead for the link and use `e.preventDefault()`?

Reply:
> You might want to switch to event handler instead for the link and use `e.preventDefault()`?

+1 we favor `preventDefault()` over inline javascript.
Looks like adding it to these to listeners will take care of it.
https://github.com/carbon-design-system/carbon-for-ibm-dotcom/blob/5eb2353f5e8bed96e3f7fa7acc3b77ba5901353e/packages/react/src/components/LocaleModal/LocaleModalRegions.js#L153-L162